src/main/java/com/dumbdogdiner/stickyapi/bukkit/particle/shapes/OutlinedSphere.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.particle.shapes;
+
+import com.dumbdogdiner.stickyapi.bukkit.particle.Orientation;
+import com.dumbdogdiner.stickyapi.bukkit.particle.Parametric;
+import com.dumbdogdiner.stickyapi.bukkit.particle.ParticleSystem;
+import com.dumbdogdiner.stickyapi.bukkit.particle.Shape;
+import org.apache.commons.lang.Validate;
+import org.bukkit.Particle;
+
+public class OutlinedSphere implements Shape {
+    @Override
+    public void draw(ParticleSystem system, Particle particle, Particle.DustOptions data) {
+
+    }
+
+    @Override
+    public void drawAbsolute(ParticleSystem system, Particle particle, Particle.DustOptions data) {
+
+    }
+
+    public enum SphereType {
+        XY,
+        YZ,
+        XZ,
+        XYZ,
+        XY_SPIRAL,
+        YZ_SPIRAL,
+        XZ_SPIRAL
+    }
+
+    public static final int MIN_DIVISIONS = 4;
+    public static final int MAX_DIVISIONS = 32;
+    public static final int MAX_RADIUS = 128;
+
+    private Parametric parametric;
+
+    public OutlinedSphere(double x, double y, double z, double r, int divisions, SphereType orientation) {
+        if (r > MAX_RADIUS) {
+            throw new IllegalArgumentException("Tried to draw sphere with absurd radius (>" + MAX_RADIUS + ")!");
+        }
+        if (r < MIN_DIVISIONS || r > MAX_DIVISIONS) {
+            throw new IllegalArgumentException("Tried to draw sphere with invalid divisions (>"
+                    + MAX_DIVISIONS + " or <" + MIN_DIVISIONS + ")!");
+        }
+
+        switch (orientation) {
```

Comment:
Why giving a sphere an orientation would make any difference to how it looks is beyond me...